src/signalrclient/signalr_default_scheduler.cpp

```diff
@@ -148,9 +148,9 @@ namespace signalr
                     // find the first callback that is ready to run, find a thread to run it and remove it from the list
                     auto curr_time = std::chrono::steady_clock::now();
                     auto it = callbacks.begin();
-                    auto found = false;
                     while (it != callbacks.end())
                     {
+                        auto found = false;
```

Comment:
There was a bug where if multiple callbacks were pending, if one of them got dispatched to a thread then the next one wasn't ready for dispatching it would still have `found == true` and get removed without being dispatched.